src/window.rs

```diff
@@ -708,9 +708,12 @@ impl Window {
 
     /// Grabs the cursor, preventing it from leaving the window.
     ///
+    /// There's no guarantee that the cursor will be hidden. You should
+    /// hide it by yourself if you want so.
```

Comment:
I know that some users reported that cursor isn't hidden automatically, I'm not sure on what platforms it's hidden though, so I decided to state it like that.

Reply:
Ah, so it's **never** hidden on Wayland? I think we should state the specifics here, I'm fine with having a separate issue to collect that information and add it later. Though I wouldn't add this statement at all then, at least for now.

Reply:
>Ah, so it's never hidden on Wayland? 

I mean that some platforms hide the cursor for you if you just issue the call to grab it, on Wayland you should call winit's method to hide a cursor yourself, since setting grab will just confine it to a window.